### DISCUSSION TITLE
conding on mobile when?

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#20387] [Mobile] [answered]
functions like coding, merging, managing and renaming the repos, when are gonna come to mobile?

**Comment by ismohamedi** [comment]
I see for now you can manage the repos, and even create new issues with Github mobile Application

**Comment by elisealix22** [comment, marked as answer]
@Nit-nit I might be misunderstanding, but merging pull requests is included today! As for coding, it's definitely something our team is thinking about and tracking. No timeline for that yet though.
